lib/stimulus_reflex/channel.rb

```diff
@@ -41,7 +42,14 @@ def receive(data)
       broadcast_message subject: "halted", data: data
     else
       begin
-        render_page_and_broadcast_morph reflex, selectors, data
+        case reflex.morph_mode
```

Comment:
Okay I‘m being a know it all here, but I really have a strong feeling „morph_mode“ should be its own class, or rather superclass. That way you could resort to polymorphism here (and possibly at other locations too) and do „tell don‘t ask“

Reply:
Respect. So, here's what's happening: I'm merging from master, then pushing, then updating the readme, then going to sleep. I'd be thrilled to wake up to a PR.

Truth is, that sounds great but I have no idea how to do what you're suggesting. If you want it, I'd love to learn.